common/src/main/java/org/spine3/samples/lobby/common/util/IdFactory.java

```diff
@@ -44,36 +44,46 @@ private IdFactory() {
      */
     public static OrderId newOrderId() {
         final String id = Identifiers.newUuid();
-        return OrderId.newBuilder().setUuid(id).build();
+        return OrderId.newBuilder()
```

Comment:
In Spine we usually have a `result` variable extracted before actual `return` statement to ease debugging.

This policy should be applied all over the codebase.